test/README.md

```diff
@@ -137,6 +139,27 @@ Once finished with the tests and the disk, and to free the ram, simply unmount t
 sudo umount /mnt/tmp
 ```
 
+**macOS**
+
+To create a 4 GiB RAM disk named "ramdisk" at `/Volumes/ramdisk/`:
+
+```bash
+diskutil erasevolume HFS+ ramdisk $(hdiutil attach -nomount ram://8388608)
+```
+
+Configure the size at the end of the command, in terms of block size
+(`8388608 / 2048 = 4096` or 4 GiB). To run the tests using the RAM disk:
```

Comment:
nit: I think the formula can be expressed a bit more useable like this:
```suggestion
(`4096 MiB * 2048 blocks/MiB = 8388608 blocks` for 4GiB). To run the tests using the RAM disk:
```

Reply:
sure, updated